### DISCUSSION TITLE
Why was MLIR-based XLA CPU backend deprecated?

### DISCUSSION THREAD

**Discussion by Joey-Ye** [openxla/xla#17659] [General]
Looking for application of MLIR in compilers I noticed a MLIR based CPU backend in older version of XLA. But it was deprecated early this year. Is there any reason why please?

**Comment by jreiffers** [comment]
There were a number of reasons, but one of them was that the GPU backend was prioritized. However, we ended up building up an MLIR pipeline for that, and there are plans to migrate CPU to this new pipeline. See xla/service/gpu/fusions/mlir.
